lib/app-routes/common/auth/keys.js

```diff
@@ -44,6 +53,33 @@ exports.register = function(server, options, next) {
                 const data = request.payload;
                 const username = data.username.toString();
                 const password = data.password.toString();
+                
+                const  recordObj = {};
+
+                /**
+                 * set recordObj attributes/values
+                 */
+                recordObj.username = data.username.toString();
+                recordObj.loginTime = new Date();
+                recordObj.ipAddr = data.ipAddress.toString();
+                recordObj.errorCode = 0;
+                recordObj.success = 1;
+
+                //Helper functions
+                /**
+                 * generate fn to call getHawkCredentials with `username`
+                 */
+                const getHawkCredentials = _.partial(
+                    authUtils.generateHawkCredentials,
+                    username
+                );
+
+                const logError = (err)=> {
+                    server.log(['error', 'login'], err);
```

Comment:
This should be:
```
server.log(['error', 'login'], err.message);
```
I know that you just copied it from the code that was already in existence, but that code was wrong...